sql/plan/insert.go

```diff
@@ -72,6 +72,9 @@ type InsertInto struct {
 
 	// FirstGenerateAutoIncRowIdx is the index of the first row inserted that increments last_insert_id.
 	FirstGeneratedAutoIncRowIdx int
+
+	// MissingValFlags marks which columns in the destination schema are expected to have default values.
+	MissingValFlags []bool
```

Comment:
could maybe pick a more specific name?`deferredDefaults`, `delayedDefaults`, `delayedNonNulls`

Reply:
i'd maybe make this a `sql.FastIntSet` also, rather than an array